src/components/accountsBottomSheet/view/accountsBottomSheetView.js

```diff
@@ -29,20 +32,42 @@ const AccountsBottomSheet = forwardRef(
       },
     }));
 
-    // _handlePressAccountTile(item)
-    const _renderAccountTile = ({ item }) => (
-      <TouchableOpacity style={styles.accountTile} onPress={() => switchAccount(item)}>
-        <View style={styles.avatarAndNameContainer}>
-          <UserAvatar username={item.username} />
-          <View style={styles.nameContainer}>
-            <Text style={styles.name}>{`@${item.username}`}</Text>
+    const _handlePressAccountTile = (item) => {
+      if (
+        item &&
+        typeof item === 'object' &&
+        Object.prototype.hasOwnProperty.call(item, 'isLoggedOut') &&
+        item.isLoggedOut
+      ) {
+        navigateToRoute(ROUTES.SCREENS.LOGIN, { username: item?.username || '' });
+      } else {
+        switchAccount(item);
+      }
+    };
+
+    const _renderAccountTile = ({ item }) => {
+      if (
+        item &&
+        typeof item === 'object' &&
+        Object.prototype.hasOwnProperty.call(item, 'isLoggedOut') &&
+        !item.isLoggedOut
+      )
+        return;
+
+      return (
+        <TouchableOpacity style={styles.accountTile} onPress={() => _handlePressAccountTile(item)}>
+          <View style={styles.avatarAndNameContainer}>
+            <UserAvatar username={item.username} />
+            <View style={styles.nameContainer}>
+              <Text style={styles.name}>{`@${item.username}`}</Text>
+            </View>
           </View>
-        </View>
-        {currentAccount.name === item.username && (
-          <Icon iconType="AntDesign" name="checkcircle" style={styles.checkIcon} size={24} />
-        )}
-      </TouchableOpacity>
-    );
+          {currentAccount.name === item.username && (
```

Comment:
There should be an options to remove listed user from logged out users... can add `X` towards end of tile to remove item